docs/docs/contribute/docs/blog.md

```diff
@@ -0,0 +1,205 @@
+  remember that many people from different organizations
+  may be contributing to this directory.
+
+    You do not need to modify the `mkdocs.yml` file for your blog.
+
+* If your blog has graphics, screen shots, YAML files, etc.
+  that will be included,
+  create an `assets` subdirectory for those files
+  in a subdirectory that has the same name
+  as the root name of your .md file.
+  For example, if your source file is named `myblog.md`
+  (which is not actually detailed enough to be a good file name),
+  you need to create a myblog/assets subdirectory.
+
+* Follow the instructions in
+  [Blog front matter](#blog-front-matter)
+  to provide the metadata required by the blog plugin.
+
+* The text that follows the `#` line until the first `##` line
+  is the introduction to your article
+  and also the abstract that is displayed on the "Blogs" landing page.
+
+Other coding notes for blogs:
+
+* Blog posts are considered part of the MkDocs NAV path.
+  This means that:
+
+    * Use the practices documented in
+      [External links and internal cross-references](code-docs.md/#external-links-and-internal-cross-references)
+      for your blog.
+    * You can use a local build to render your blog locally as you write.
+      See
+      [Build documentation locally](local-building.md)
+      for details.
+    * The `readthedocs.build` preview associated with your PR
+      contains the rendered version of your blog
+      so that you and your reviewers can see it.
+
+## Blog front matter
+
+The blog plugin requires some information to manage the blog.
+This is specified as part of the file's metadata.
+Here we document the fields that are required.
+Additional fields can be added; see the
+[blog plugin documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#writing-your-first-post)
+for more information.
+
+```md
+---
+date: YYYY-MM-DD
+authors: [<GitHub-UserID>]
+description: >
+  <Brief description of this blog, all in one source line>
+categories:
+  - <cat-1>
+  - <cat-2>
+  - ...
+comments: true
+---
+
+# <blog title>
+```
+
+### Blog front matter fields
+
+* **date** -- Date when blog was most recently posted
+* **authors** -- Author of this blog,
+  identified by the Github User ID that is used as the key
+  in the `docs/blog/.authors.yml` file.
+  This is used to generate the author information
+  that is displayed.
+* **description: >** -- Brief description
+* **categories:** -- Keywords used to generate entries in the "Categories"
+  section of the "Blogs" landing page.
+  Set as many categories as appropriate and use reasonable terminology.
+  If an existing category matches a category you want for your blog,
+  be sure to match that terminology exactly.
+  For example, "Installation" is an existing category
+  so it would be inappropriate to define "Installing" as a category.
+* **comments: true**
+* **blog-title** -- The title that is displayed for your blog,
+  coded as a level 1 (`#`) header.
```

Comment:
```suggestion
* **blog-title** -- The title that is displayed for your blog post.
  Only use this, if you want the blog post to have a different title compared to
  what is coded as a level 1 (`#`) header in your post.
```

Reply:
I don't think this is right. I was using `blog-title` as the text for the `#` line so it makes no sense to say this is populated only if you want a title that is different from what is in the # line.

Reply:
I thought there's a metadata field for that but looking at the docs, there's none. Only for the slug which is used in the URL.
So I think we could fully remove this since there's no field named `blog-title`

Reply:
Ah, so you were viewing `blog-title` as a metadata field.  I reworked the material a little bit -- we need to state that the title is coded as an H1 and explain about the text that is both intro and abstract but it does look better this way, I think.

Reply:
yea i considered it as part of the front matter since it was explained there 😄 
but the explanation looks good now, i like it!